app/cronjob/trailingTrade/step/determine-action.js

```diff
@@ -65,13 +65,17 @@ const canSell = data => {
     symbolInfo: {
       filterMinNotional: { minNotional }
     },
+    symbolConfiguration: {
+      buy: { lastBuyPriceRemoveThreshold }
+    },
     baseAssetBalance: { total: baseAssetTotalBalance },
     sell: { currentPrice: sellCurrentPrice, lastBuyPrice }
   } = data;
 
   return (
     lastBuyPrice > 0 &&
-    baseAssetTotalBalance * sellCurrentPrice > parseFloat(minNotional)
+    baseAssetTotalBalance * sellCurrentPrice > parseFloat(minNotional) &&
```

Comment:
I have tested manually, this line does not work. You gotta remove the below line because the bot is now relying on the `lastBuyPriceRemoveThreshold`

```
baseAssetTotalBalance * sellCurrentPrice > parseFloat(minNotional) &&
```

You can see from the below screenshot, the bot didn't remove the price, it still determines the action as `sell-wait`.

![image](https://user-images.githubusercontent.com/5715919/121687521-ca0c2280-cb05-11eb-8a63-42e340cedbe0.png)